<launch>
    <node pkg="obstacle_detection" type="obstacle_detection_node" name="obstacle_detection_node">
        <param name="box_height" type="double" value="5" />
        <param name="box_width" type="double" value="2" />

        <param name="cluster_tolerance" type="double" value="0.1" />
        <param name="min_cluster_size" type="int" value="4" />
        <param name="max_cluster_size" type="int" value="400" />

        <param name="hue_max" type="int" value="100"/>    <!-- green -->
        <param name="hue_min" type="int" value="70"/>    <!-- green -->
        <param name="sat_max" type="int" value="255"/>
        <param name="sat_min" type="int" value="50"/>
        <param name="val_max" type="int" value="255"/>
        <param name="val_min" type="int" value="0"/>

        <param name="focal_length_x" type="double" value="598.66311"/>
        <param name="focal_length_y" type="double" value="596.74570"/>
        <param name="focal_center_x" type="double" value="443.39763"/>
        <param name="focal_center_y" type="double" value="267.92429"/>

        <param name="show_source" type="bool" value="true"/>
	    <param name="show_binary" type="bool" value="false"/>
    </node>
</launch>
